README.md

```diff
@@ -34,6 +34,12 @@ If you discover a security vulnerability, contact the development team by e-mail
 
 Alloy Core defines low-level utilities for basic Java use patterns
 
+### alloy-collections
```

Comment:
is this part of the readme valid anymore?

Reply:
Nope, let me fix that

Reply:
@nickavv FIxed, carry on